ToolbarActionButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { Platform, Pressable, StyleSheet, Text } from "react-native";
+import React from "react";
+
+export const ToolbarActionButton = ({
+  label,
+  onPress,
+}: ToolbarActionButtonProps) => {
+  const formattedLabel = Platform.OS === "ios" ? label.toUpperCase() : label;
```

Comment:
I think this is mixed up, right? Normally iOS button bars are not uppercased!

Reply:
Oh right 😄 I inverted it 👍